packages/Radio/src/components/Group/Group.js

```diff
@@ -60,7 +60,6 @@ function Group(props) {
             canDeselect,
             name: child.props.name || name,
             ...childKey,
-            ...moreGroupProps,
```

Comment:
This is an unrelated cleanup fix, but will investigate if this change has any repercussions in our consumer applications